token/program-2022/src/extension/confidential_transfer/account_info.rs

```diff
@@ -0,0 +1,276 @@
+#[cfg(not(target_os = "solana"))]
```

Comment:
nit for simplicity: does it make sense to mark this whole file with `#![cfg(not(target_os = "solana")))]`? and not have all of the individual annotations? or even mark the module declaration in `confidential_transfer/mod.rs` with the cfg?

Reply:
Ah yes, this will be much simpler! I marked the module declaration to be `#![cfg(not(target_os = "solana")))]`.